components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -325,15 +325,17 @@ void BravePrefProvider::UpdateCookieRules(ContentSettingsType content_type,
         ContentSettingsPattern::FromString(kGoogleAuthPattern),
         ContentSettingsPattern::Wildcard(),
         base::Value::FromUniquePtrValue(
-            ContentSettingToValue(CONTENT_SETTING_ALLOW)));
+                         ContentSettingToValue(CONTENT_SETTING_ALLOW)),
+                     base::Time(), SessionModel::Durable);
```

Comment:
I'm pretty sure that `Durable` is not correct here. All of these settings are built from other settings and they shouldn't be persisted for that reason.

Reply:
I need to look at the chromium code for this to see what makes the most sense

Reply:
I guess default rules are also `SessionModel::Durable` so I think this is ok